### DISCUSSION TITLE
Entity errors?

### DISCUSSION THREAD

**Discussion by bouchajd** [jmcollin78/solar_optimizer#33] [General]
Hi everyone - I was quite excited to see this project as I'm trying to get my tesla set up to charge with excess solar - seems exactly what i was looking for. However I'm having issues to get this working - some errors on startup even though I set up per the instructions - and I copied the example for charging a tesla but I'm struggling to get it working - any advice on how to get this working withe teslamate? 

I'm seeing the following errors in my home assistant notifications:

Logger: homeassistant.components.sensor
Source: components/sensor/__init__.py:570
integration: Sensor ([documentation](https://www.home-assistant.io/integrations/sensor), [issues](https://github.com/home-assistant/core/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+sensor%22))
First occurred: 2:11:07 PM (3 occurrences)
Last logged: 2:11:07 PM
•	Entity sensor.total_power (<class 'custom_components.solar_optimizer.sensor.SolarOptimizerSensorEntity'>) is using state class 'total' which is impossible considering device class ('power') it is using; expected None or one of 'measurement'; Please update your configuration if your entity is manually configured, otherwise create a bug report at https://github.com/jmcollin78/solar_optimizer/issues
•	Entity sensor.power_production (<class 'custom_components.solar_optimizer.sensor.SolarOptimizerSensorEntity'>) is using state class 'total' which is impossible considering device class ('power') it is using; expected None or one of 'measurement'; Please update your configuration if your entity is manually configured, otherwise create a bug report at https://github.com/jmcollin78/solar_optimizer/issues
•	Entity sensor.power_production_brut (<class 'custom_components.solar_optimizer.sensor.SolarOptimizerSensorEntity'>) is using state class 'total' which is impossible considering device class ('power') it is using; expected None or one of 'measurement'; Please update your configuration if your entity is manually configured, otherwise create a bug report at https://github.com/jmcollin78/solar_optimizer/issues

**Comment by jmcollin78** [comment]
Hello,

You can forget those errors which are only warningsand doesn't have any impact. There is already an issue opened to fix that.
